std/bigint.d

```diff
@@ -421,6 +422,31 @@ public:
     `58105600`
         ));
     }
+// https://issues.dlang.org/show_bug.cgi?id=10565
+@safe unittest
+{
+    // Test cases from the issue
+    BigInt a = BigInt("0");
+    BigInt b = BigInt("-0");
+    BigInt c = BigInt("0") * -1;
+    BigInt d = BigInt("0") * -42;
+    BigInt e = BigInt("0"); e *= -1;
+    BigInt f = BigInt(c);
+    BigInt g = BigInt("0") * cast(byte) -1;
+    BigInt h = BigInt("0"); h *= BigInt("-1");
+    BigInt i = BigInt("0"); i -= 2 * i;
+    BigInt j = BigInt("0"); j = -j;
+    
```

Comment:
```
Check for trailing whitespace
grep -nr '[[:blank:]]$' $(find etc std -name '*.d'); test $? -eq 1
std/bigint.d:439:
```

I'm not entirely sure why the style checker fails here, there doesn't appear to be extra whitespace.